### PR TITLE
Feature/after demo changes

### PR DESCRIPTION
**Pull request by taimoormalik97**

### CODE REVIEW

File: app/models/doctor.rb [hunk 1]
```diff
@@ -18,7 +18,7 @@ def self.search_doctors(pattern)
     if pattern.blank?  # blank? covers both nil and empty string
       all
     else
-      where('name LIKE ?', "%#{pattern}%")
+      where('name LIKE ?', "%#{pattern}%").first(TOKENINPUT_LIMIT)
```
Comment (sulemanuzair) [inline app/models/doctor.rb]: use limit

Comment (sulemanuzair) [reply app/models/doctor.rb]: change in other occurences as well in this PR

Comment (taimoormalik97) [reply app/models/doctor.rb]: done

File: app/views/appointments/index.html.erb [hunk 2]
```diff
@@ -40,15 +40,14 @@
                 <td><%= convert_to_twelve_hour_format(availability.end_slot) %></td>
                 <% status_colors = status_colors(appointment) %>
                 <% unless current_user.doctor? %>
-                  <td><%= appointment.doctor.name %></td>
+                  <td><%= appointment.doctor.name.titleize %></td>
```
Comment (sulemanuzair) [inline app/views/appointments/index.html.erb:43]: why using titlize ?

**Pull request merged**